src/go/util/mm/minimega.go

```diff
@@ -148,6 +149,7 @@ func (this Minimega) GetVMInfo(opts ...Option) VMs {
 		vm.UUID = row["uuid"]
 		vm.Host = row["host"]
 		vm.Name = row["name"]
+		vm.CCActive, _ = strconv.ParseBool(row["cc_active"])
```

Comment:
I have found the `cc_active` column from `vm info` to be unreliable. I've seen cases where a miniccc agent in a VM has failed for some reason and `cc_active` still reports true. I would suggest using the `mm.IsC2ClientActive` function instead, since it uses the more reliable `cc client` command to determine if cc is active for a VM.

Reply:
Let me know your thoughts on 280b16c135374abee536ec7963d3a08b64077b25. I found there was a fair bit of slow down if I ran `IsC2ClientActive` for every vm in an experiment, so I have it only running for single vm calls to `GetVMInfo` right now. It's possible I'm just not providing the right options and that's what is slowing things down.

The cc active flag is currently being used in the vm info modal and is displayed as text and used for disabling the mount button. I previously made the info modal make an extra backend call to get the latest info for its vm, so it should be getting the `IsC2ClientActive` value.

Reply:
@jacdavi I'll take a look at this ASAP and see if I can't come up with a way to speed up client status requests for multiple VMs at the same time. I have an idea of how to do so.

Reply:
I think I came up with a way to speed this up. It seems to be working as expected.